cl/simple_pages/templates/components.html

```diff
@@ -1,7 +1,17 @@
 {% extends "base.html" %}
 
+{% load static tailwind_tags %}
```

Comment:
This loads tailwind just in this file, not in `base.html`. I assume this is better as an initial test